tests/functional/customer/test_alert.py

```diff
@@ -27,21 +30,19 @@ def test_can_create_a_stock_alert(self):
         form.submit()
 
         alerts = ProductAlert.objects.filter(user=self.user)
-        self.assertEqual(1, len(alerts))
+        assert len(alerts) == 1
```

Comment:
I'd suggest we avoid unnecessary changes like this, or make them in a separate PR - it makes the diff on this PR much bigger, and harder to see what the functional changes are. From what I can see we're just changing to pytest-style assertions here, but the actual logic is unchanged.

Reply:
Yes, you are right - the actual logic is unchanged.

Will do changes like this in separate PR in the future. Should I remove them from this PR, or we can keep them?